src/client/Wrapper.js

```diff
@@ -73,6 +78,42 @@ export default class Wrapper extends React.PureComponent {
     this.props.getTrendingTopics();
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.locale !== this.props.locale) {
+      this.setState({ locale: this.getAvailableLocale(nextProps.locale) });
```

Comment:
Does this actually work for anyone @Sekhmet @bonustrack ? I took the time and ran this on my local..

<img width="410" alt="screen shot 2017-12-15 at 5 26 28 am" src="https://user-images.githubusercontent.com/5067716/34037935-8dd71bc0-e158-11e7-9cd4-dbd7a1e55588.png">

and keep seeing these errors when Im switching languages